.pullaprove.yml

```diff
@@ -0,0 +1,14 @@
+approve_by_comment: true
+approve_regex: (?i)^Approved
```

Comment:
Maybe something close to the one on the [docs website](http://docs.pullapprove.com/yaml/approve_regex/)?
```yaml
approve_regex: '(?i)^(approved|looks good|:shipit:|:\+1:)'
```
Then "Looks good to me." would for example match as well.